### DISCUSSION TITLE
tail -f /var/file.log | grep 'whatever' | notify

### DISCUSSION THREAD

**Discussion by gprime31** [projectdiscovery/notify#113] [Q&A] [answered]
tail -f /var/file.log | grep 'whatever' | notify
is there a way to make it work?

**Comment by riramar** [comment]
Do you know why this is not working?
I installed notify just to work with grep but I didn't know it doesn't work.
I do a little bit of coding in golang so maybe I can try to help on that.

**Comment by riramar** [comment]
I've added support in my fork and sent an PR.
You can check my changes here https://github.com/riramar/notify.

**Comment by MetzinAround** [comment, marked as answer]
https://github.com/projectdiscovery/notify/pull/195

PR was closed as not planned, will also close this discussion.
